doc/configuration.rst

```diff
@@ -74,7 +74,7 @@ This allows identification through hot-plugging or rebooting.
 
    USBSerialPort:
      match:
-       'ID_SERIAL_SHORT': 'P-00-00682'
+       ID_SERIAL_SHORT: 'P-00-00682'
```

Comment:
This should be quoted and needs a `@` as well (see below).

Reply:
On the examples I was working on the code would crash if it had the ''
I didn't try with the @ tho. 
However, there are code examples (commited by you :P) that do not use the quotes... 
https://github.com/labgrid-project/labgrid/blob/a6571a44501df93c649fafe315d75097d95d4941/examples/barebox/local-usb.yaml

My wider issue is that the pattern seems to be different on every file I checked, which made it hard for me to tell what the expected code was.

Reply:
The output from the labgrid-suggest tool should be correct for your local system.